docs/guide/README.md

```diff
@@ -0,0 +1,12 @@
+Assets
+======
+
+<p align="justify">An asset in Yii is a file that may be referenced in a Web page. It can be a CSS file, a JavaScript file, an image or video file, etc. Assets are located in Web Accessible directories and are directly served by Web servers.</p>
```

Comment:
There's no need to wrap everything with `<p align="justify">`.

Reply:
Don't we have assets originally located in non-public directories? The ones we should publish before referencing from web page.